openstack/tools/openstack_regression_tests_runner.sh

```diff
@@ -90,14 +90,20 @@ FIP_MIN_ABC=${FIP_MIN%.*}
 FIP_MIN_D=${FIP_MIN##*.}
 FIP_MIN=${FIP_MIN_ABC}.$(($FIP_MIN_D + 64))
 
-CIDR_OAM=$(openstack subnet show subnet_${OS_USERNAME}-psd -c cidr -f value)
```

Comment:
I didn't realize the code was duplicated. Could be a good future idea to move this code to a function in a single file that is used by both scripts if there is nothing preventing that. Something for the future improvements, not now.

Reply:
+1